### PR TITLE
Add banner editing to EditProfile

### PR DESCRIPTION
**Pull request by GnomedDev**

### CODE REVIEW

File: src/builder/edit_profile.rs [hunk 1]
```diff
@@ -64,6 +66,20 @@ impl EditProfile {
         self.username = Some(username.into());
         self
     }
+
+    /// Sets the banner of the current user.
+    ///
+    /// This takes the raw image data as base64 encoding is handled internally.
+    pub fn banner(mut self, banner: &[u8]) -> Self {
```
Comment (arqunis) [inline src/builder/edit_profile.rs]: Shouldn't this use `CreateAttachment` just like `EditProfile::avatar`? If not, the documentation should mention that `banner` should be PNG data.

Comment (GnomedDev) [reply src/builder/edit_profile.rs]: I don't know how the current CreateAttachment stuff works, I'll take a look.

Comment (GnomedDev) [reply src/builder/edit_profile.rs]: This seems the wrong way to use it, since it doesn't require a filename, but meh that's not this PR's job to fix.

Comment (GnomedDev) [reply src/builder/edit_profile.rs]: Sorted this out.

**Review by arqunis — APPROVED** [review]

**Pull request merged**